pkg/kubelet/kubelet_node_status.go

```diff
@@ -319,16 +321,57 @@ func (kl *Kubelet) syncNodeStatus() {
 	}
 }
 
+// A notifier to make sure logs express backoff state when the kubelet is in trouble.
+var backoffNotifierRunning = false
+
+func (kl *Kubelet) backoffNotifier() {
```

Comment:
+1 to what @dchen1107 wrote. I'm fine-ish with the change for existing Nodes, but it should have a special case for retrying setting the first NodeStatus (i.e. registering to Master).